components/dashboard/src/teams/TeamBilling.tsx

```diff
@@ -232,25 +243,31 @@ export default function TeamBilling() {
                                         </span>
                                     ))}
                                 </div>
-                                <div className="flex-grow flex flex-col items-end justify-end"></div>
+                                <div className="flex-grow flex flex-col items-stretch justify-end"></div>
                             </div>
                         </Card>
                         {!teamSubscription ? (
-                            <SolidCard>
+                            <SolidCard className="w-72 h-64">
                                 <div className="w-full h-full flex flex-col items-center justify-center">
                                     <Spinner className="h-5 w-5 animate-spin" />
                                 </div>
                             </SolidCard>
                         ) : (
-                            <SolidCard>
+                            <SolidCard className="w-72 h-64">
```

Comment:
**thought**: Hoping in the future we can drop these by introducing component variants that include card size, etc. instead of hard-coding sizes like this. We'll get there!

Reply:
There are ways to achieve this with a single component, e.g. by using:
- `min-height` on the card (to allow it to grow arbitrary large depending on its content)
- `items-stretch` on the card container (to make all the cards the same height, even if one card has a lot of content but the others don't)

Reply:
Also, it's good to keep in mind that Tailwind is so nice because it allows directly "hard-coding" style in a nice way (as opposed to hiding style under multiple levels of abstractions, making them really hard to adjust afterwards).

Reply:
> Long-term, to keep the component flexible I'd love to introduce a stepper property to increase the width or height without using Tailwind's classes. 🧡

But, even better would be if the Cards always look perfectly-sized, depending on their contents, _without_ needing a stepper or Tailwind classes, right? 😇 ("Default correct" is better than "lots of knobs and switches")

Reply:
> Also, it's good to keep in mind that Tailwind is so nice because it allows directly "hard-coding" style in a nice way

I'm also in favor of duplication over abstraction as it comes with multiple benefits, but in the long run it makes sense to abstract _some_ component aspects to help a) avoid running into code reviews nitpicking dimensions or other attributes, b) avoid requesting UX reviews when possible to avoid, c) feeling more confident about the product UX, and more.

However, this issue will become more relevant as these components mature and are widely used. For now, as long as we have two component instances of the solid card component, any approach would suffice and does not worth keeping this PR from being merged. 🤝 

> But, even better would be if the Cards always look perfectly-sized, depending on their contents, without needing a stepper or Tailwind classes, right?

Debatable! 🙂 

In most cases, the card component would need to use a specified width or height, which could be challenging to maintain when having multiple cards side-by-side that from a design perspective have to use fixed and specific dimensions regardless if some contents are missing. For example, see editor preferences, etc.

However, there's still need for having flexible card layouts without fixed width that fill the parent container on smaller viewports, etc.